pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -39,26 +38,43 @@ import (
 	v1 "k8s.io/api/apps/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"text/template"
 )
 
-var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mgr")
+var (
+	logger             = capnslog.NewPackageLogger("github.com/rook/rook", "op-mgr")
+	prometheusRuleName = "prometheus-ceph-vVERSION-rules"
```

Comment:
Shall we get rid of the ceph version in the rules name? It doesn't seem relevant anymore, right?
```suggestion
	prometheusRuleName = "prometheus-ceph-rules"
```

Reply:
Right, removed